build race with drmemfuncs vs event_strings

The cross-arch tests linux.execve{32,64} are failing on CDash.  E.g.:
http://dynamorio.org/CDash/testDetails.php?test=205911&build=41693

Looking more closely it's a build race b/c the newly added memmove.c to drmemfuncs did not come with an explicit dep on event_strings:
```
Scanning dependencies of target drmemfuncs
[  0%] Building ASM object core/CMakeFiles/drmemfuncs.dir/arch/x86/memfuncs.asm.o
[  0%] Building C object core/CMakeFiles/drmemfuncs.dir/lib/memmove.c.o
In file included from /work/dr/nightly/src/core/lib/../globals.h:345:0,
                 from /work/dr/nightly/src/core/lib/memmove.c:44:
/work/dr/nightly/src/core/lib/../utils.h:1944:10: fatal error: event_strings.h: No such file or directory
 #include "event_strings.h"
          ^~~~~~~~~~~~~~~~~
```

--- a/core/CMakeLists.txt
+++ b/core/CMakeLists.txt
@@ -89,6 +89,7 @@ if (UNIX AND X86)
   add_asm_target(arch/x86/memfuncs.asm memfuncs_asm_src memfuncs_asm_tgt
     "_memfuncs" "" "${asm_deps}")
   add_library(drmemfuncs STATIC ${memfuncs_asm_src} lib/memmove.c)
+  add_gen_events_deps(drmemfuncs)
 endif ()
 
 # i#1409: to share core libc-ish code with non-core, we use the "drlibc" library.

